corehq/apps/domain/templates/domain/email/domain_invite.txt

```diff
@@ -2,7 +2,7 @@
 {% blocktrans %}Hey there,{% endblocktrans %}
 
 {% blocktrans %}
-    {{ inviter }} has invited you to join the {{ domain }} project space at CommCare HQ.
+    {{ inviter }} has invited you to join the {{ domain }} project at CommCare HQ.
```

Comment:
Just curious what goes into deciding if the text should be project vs project space? I'm sure this is an age old discussion, but I've been operating with "project" being the organization vs "project space" being the user facing language for domain. But from this perspective, I totally understand "project" sounding better here.

Reply:
I don't know that I've argued about project vs project space before. I picked "project" here because that seems to be what we use within HQ itself: the project switcher in the top right (and the page it goes to), the "My Projects" page under account settings, that sort of thing.